src/app/widgets/got/list-of-characters/character/character.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+import { Character } from "../../interfaces/Character";
+
+@Component({
+  selector: "app-character",
+  templateUrl: "./character.component.html",
+  styleUrls: ["./character.component.scss"],
+})
+export class CharacterComponent {
+  @Input() character!: Character;
+  @Output() detailBtnClicked = new EventEmitter<Character>();
+  @Output() favouriteBtnClicked = new EventEmitter<Character>();
```

Comment:
to chyba też nie potrzebne bo widzę że jest wykomentowany kod do obsługi tego zdarzenia